src/main/scala/com/twitter/finatra/http/SwaggerRouteDSL.scala

```diff
@@ -2,17 +2,18 @@ package com.twitter.finatra.http
 
 import com.jakehschwartz.finatra.swagger.FinatraSwagger
 import com.twitter.finagle.http.{Request, Response, RouteIndex, Status}
-import io.swagger.models.{Operation, Swagger}
+import io.swagger.v3.oas.models.PathItem.HttpMethod
+import io.swagger.v3.oas.models.{OpenAPI, Operation}
 
 /**
   * To work around the accessibility of RouteDSL, this class is in "com.twitter.finatra.http" package
   */
 object SwaggerRouteDSL {
-  implicit def convert(dsl: RouteDSL)(implicit swagger: Swagger): SwaggerRouteDSL = new SwaggerRouteDSLWrapper(dsl)(swagger)
+  implicit def convert(dsl: RouteDSL)(implicit swagger: FinatraSwagger): SwaggerRouteDSL = new SwaggerRouteDSLWrapper(dsl)(swagger)
 }
 
 trait SwaggerRouteDSL extends RouteDSL {
-  implicit protected val swagger: Swagger
+  implicit protected val swagger: FinatraSwagger
```

Comment:
consider renaming for consistency:
implicit protected val swagger: FinatraSwagger

to 

implicit protected val finatraSwagger: FinatraSwagger